src/dynamic-checkout/utils/translations.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference path="../references.ts" />
+
+module ProcessOut {
+  export class Translations {
+    static getText(key: string, locale: string) {
+      switch (locale) {
```

Comment:
Why don't we just build a map / object for this?
```javascript
const locales = {
  "en": en,
  "fr": fr,
  "pl": pl
}
```

And then just retrieve it from the object?
```javascript
const defaultLocale = "en"
static getText(key: string, locale: string) {
  const keys = locales[locale] ?? locales[defaultLocale];
  return keys[key];
}
```

Also, what happens if we find no given key? Should we return some default "placeholder" maybe? Right now, it'd probably return just "undefined"?

Reply:
Yeah map is better approach, I will fix it. When key is not found I will just return empty string